app/models/alchemy/page.rb

```diff
@@ -405,5 +413,18 @@ def update_published_at
       self.published_at = Time.now
     end
 
+    # Creates a node for this page
+    #
+    # If the page has a parent and the parent a node then the node will placed as child of parents node
```

Comment:
Metrics/LineLength: Line is too long. [103/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)